html/functions.php

```diff
@@ -90,3 +90,34 @@ function createTweetreply($text, $user_id, $reply_id)
     return $stmt->execute();
 }
 /* 返信課題はここからのコードを修正しましょう。 */
+
+function createfavorite($member_id, $post_id)
```

Comment:
```suggestion
function createfavorite($member_id, $post_id) {
```

こちらもコードの記述の話で細かい部分ですが、少なくとも1つのファイルの中で記述は統一しましょう。

記述が混じっています。
ちなみに、この手の指摘はlint（りんと、という）というツールを使って自動で指摘・修正することもあります。

興味があれば調べてみてください。


```php
function sampleMethod() {
}

だったり、

function sampleMethod()
{
}

だったり
```